Makefile

```diff
@@ -49,7 +55,15 @@ docker-build-manager: IMAGE_TAG=$(MANAGER_IMAGE)
 docker-build-ebpf:
 	docker build --platform linux/$(OS_ARCH) --build-arg ARCH=$(OS_ARCH) -t ebpf-builder-$(OS_ARCH) -f bin/ebpf-builder/Dockerfile ./bin/ebpf-builder/
 	-rm -r bin/injector/ebpf/
-	docker run --rm -v $(shell pwd):/app ebpf-builder-$(OS_ARCH)
+# create a dummy container with volume to store files
```

Comment:
Need to use this approach due to circleci:
* https://circleci.com/docs/building-docker-images/#mounting-folders